model/patch_lifecycle.go

```diff
@@ -409,6 +409,11 @@ func FinalizePatch(ctx context.Context, p *patch.Patch, requester string, github
 			},
 		)
 	}
+	if p.IsChild() {
```

Comment:
I think we need to call this earlier in the function. The parameters get read into the version [here](https://github.com/evergreen-ci/evergreen/blob/38686c59913a76522b89642212af48defb6d6628/model/patch_lifecycle.go#L315).